benchalerts/parse_conbench.py

```diff
@@ -46,35 +85,58 @@ def benchmarks_with_z_regressions(
 def regression_summary(
     comparisons: List[RunComparison], warn_if_baseline_isnt_parent: bool
 ) -> str:
-    """Generate a Markdown summary of what happened regarding regressions."""
+    """Generate a Markdown summary of what happened regarding errors and regressions."""
     sha = comparisons[0].contender_info["commit"]["sha"][:8]
+    errors = benchmarks_with_errors(comparisons)
+    regressions = benchmarks_with_z_regressions(comparisons)
+    summary = ""
+
+    if errors:
+        summary += _clean(
+            """
+            ## Benchmarks with errors
+
+            These are errors that were caught while running the benchmarks. You can
+            click the link next to each case to go to the Conbench entry for that
+            benchmark, which might have more information about what the error was.
+            """
+        )
+        previous_run_id = ""
+        for run_id, run_url, name, error_url in errors:
+            if run_id != previous_run_id:
+                summary += f"\n\n- Run ID [{run_id}]({run_url})"
+                previous_run_id = run_id
+            summary += f"\n  - `{name}` ([link]({error_url}))"
```

Comment:
Very minor: Could we have the name be the link itself (so we don't need "(link)" after? Or could we use the phrase "view on conbench" instead of "link" if we want the link to be separate from the name? I think that'll look a little less redundant

Reply:
I did this originally because I liked using `preformatted` text for what is probably autogenerated & technical vernacular in the case names. But glancing again at what Arrow uses, I actually don't think it would look too bad as hyperlinks. Let me try that out and see what you think.